src/modules/routes.ts

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const router = express.Router();
+
+const {
+    getPurchases,
+    createPurchase,
+    updatePurchaseInfo,
+    deletePurchase
+} = require('./controllers');
+
+router.get('/', getPurchases);
+router.post('/createPurchase', createPurchase);
+router.patch('/updatePurchase', updatePurchaseInfo);
+router.delete('/deletePurchase', deletePurchase);
```

Comment:
Just so that you know, REST has certain rules in endpoints naming - you should have called your endpoint like this:
GET /purchases
POST /purchases
PATCH /purchases/:id
DELETE /purchases/:id